test/viaq-data-model.sh

```diff
@@ -98,7 +98,7 @@ cat > $cfg <<EOF
     undefined4 undefined4
     undefined5 undefined5
   </record>
-  remove_keys CONTAINER_TAG
+  remove_keys CONTAINER_TAG,logtag
```

Comment:
I've pushed this patch to https://github.com/openshift/origin-aggregated-logging/pull/1350, which adds missing keep_partial_key.  Since the default value is false, logtag is not supposed to be kept in the record.
https://github.com/openshift/origin-aggregated-logging/pull/1350/commits/d7d0ba7384f44134a1572d3e385a013cb96986ff